profile/README.md

```diff
@@ -0,0 +1,132 @@
+# Welcome, Apes!
+
+We are [ApeWorX LTD](https://apeworx.io), a collective of infrastructure engineers, smart contract devs, and Web3-curious folks,
+building the next generation of Python tooling and infrastructure to support your next great idea.
+Our suite of open source projects and developer tools, educational conten, and hosted services aims to guide you along your
+journey from idea to production, supporting you beyond the launch to ensure you build the highest quality projects.
+
+Come chat with us online on Discord or X (twitter) to stay up to date on new releases, plugins, and tutorials!
+
+[![Discord chat][discord-badge]][discord-url]
+[![Twitter][twitter-badge]][twitter-url]
+
+## Our Projects
+
+### Ape
+
+[Ape Framework](https://apeworx.io/framework) is an easy-to-use Web3 development tool.
+Users can compile, test, and interact with smart contracts all in one command line session.
+With our [modular plugin system](https://docs.apeworx.io/ape/stable/userguides/quickstart.html#plugin-system),
+Ape supports multiple smart contract languages, chains, developer tools/services, and more!
+
+### Silverback
+
+[Silverback](https://apeworx.io/silverback) lets you create and deploy your own Python bots that respond to on-chain events.
+The Silverback library leverages Ape Framework as well as it’s ecosystem of plugins and packages to enable you to develop
+simple-yet-sophisticated automated applications that can listen and respond to live chain data in real time.
+To get started developing with Silverback, check out our [Quickstart Userguide](https://docs.apeworx.io/silverback/stable/userguides/quickstart)
+or visit [https://silverback.apeworx.io](https://silverback.apeworx.io) to sign up for the Silverback cloud-hosted application platform.
+
+## Resources
+
+Visit our [technical documentation hub](https://docs.apeworx.io) to get a deeper understanding of Ape, Silverback and our other OSS packages.
+
+<!-- Read our [academic platform](https://academy.apeworx.io/) will help you master Ape Framework with tutorials and challenges. -->
+
+## Contributing
```

Comment:
Should this part just link to the CONRTIBUTING guide? We can make a some of these edits there